pkg/manager/internal.go

```diff
@@ -237,6 +246,8 @@ func (cm *controllerManager) Start(stop <-chan struct{}) error {
 		go cm.serveMetrics(cm.internalStop)
 	}
 
+	go cm.startHA()
```

Comment:
it's not clear what HA means -- we should say "LeaderElection" or something instead (e.g. `NeedsLeaderElection`, `startWithLeaderElection`, etc).